site/content/contribute/more-info/webapp/_index.md

```diff
@@ -10,28 +10,25 @@ aliases:
 
 The Mattermost web app is written in JavaScript using {{< newtabref href="https://facebook.github.io/react/" title="React" >}} and {{< newtabref href="https://redux.js.org/" title="Redux" >}}.
 
-It is powered by [mattermost-redux]({{< ref "/contribute/more-info/webapp/redux" >}}) which handles the majority of the logic, client-side storage and server communication.
-
-Please note that issues associated with the Mattermost web app are found only in the {{< newtabref href="https://github.com/mattermost/mattermost-server" title="Mattermost Server repo" >}}
 ## Repository
 
-https://github.com/mattermost/mattermost-webapp
+It is located in the `webapp` directory of the {{< newtabref href="https://github.com/mattermost/mattermost-server" title="main Mattermost monorepo" >}}.
+
+https://github.com/mattermost/mattermost-server/tree/master/webapp
 
 ## Help Wanted
 
-{{< newtabref href="https://mattermost.com/pl/help-wanted-mattermost-webapp/" title="Find help wanted tickets here." >}}
+{{< newtabref href="https://mattermost.com/pl/help-wanted-mattermost-webapp/" title="Find help wanted tickets here" >}}.
+
+## Package structure
```

Comment:
A lot of these docs still pertain to `channels`/the main web app, but I wanted to call out the other packages here as we start to hopefully move towards treating it as a monorepo in the docs